hack/generate/csv.sh

```diff
@@ -125,17 +125,20 @@ cp "$template" "$target"
 for name in "${images[@]}"; do
   echo "Image: ${name} -> ${images_addresses[$name]}"
   add_related_image "$target" "IMAGE_${name}" "${images_addresses[$name]}"
-  add_downstream_operator_deployment_image "$target" "IMAGE_${name}" "${images_addresses[$name]}"
-  add_upstream_operator_deployment_image "$target" "IMAGE_${name}" "${images_addresses[$name]}"
+  add_downstream_operator_deployment_env "$target" "IMAGE_${name}" "${images_addresses[$name]}"
+  add_upstream_operator_deployment_env "$target" "IMAGE_${name}" "${images_addresses[$name]}"
 done
 
 # don't add Kafka image overrides to upstream operator
 for name in "${kafka_images[@]}"; do
   echo "kafka Image: ${name} -> ${kafka_images_addresses[$name]}"
   add_related_image "$target" "KAFKA_IMAGE_${name}" "${kafka_images_addresses[$name]}"
-  add_downstream_operator_deployment_image "$target" "KAFKA_IMAGE_${name}" "${kafka_images_addresses[$name]}"
+  add_downstream_operator_deployment_env "$target" "KAFKA_IMAGE_${name}" "${kafka_images_addresses[$name]}"
 done
 
+# Add Knative Kafka version to the downstream operator
+add_downstream_operator_deployment_env "$target" "KNATIVE_EVENTING_KAFKA_VERSION" "$(metadata.get dependencies.eventing_kafka)"
```

Comment:
Don't know the function for fetching the metadata, but looks ok for me. (guess it's battle-tested already elsewhere)